pkg/apiserver/database/gorm/odata.go

```diff
@@ -879,6 +879,96 @@ var schemaMetas = map[string]odatasql.SchemaMeta{
 			},
 		},
 	},
+	scanEstimationSchemaName: {
+		Table: "scan_estimations",
+		Fields: odatasql.Schema{
+			"id":        odatasql.FieldMeta{FieldType: odatasql.PrimitiveFieldType},
+			"startTime": odatasql.FieldMeta{FieldType: odatasql.PrimitiveFieldType},
+			"endTime":   odatasql.FieldMeta{FieldType: odatasql.PrimitiveFieldType},
+			"assetIDs": odatasql.FieldMeta{
+				FieldType: odatasql.CollectionFieldType,
+				CollectionItemMeta: &odatasql.FieldMeta{
+					FieldType: odatasql.PrimitiveFieldType,
+				},
+			},
+			"state": odatasql.FieldMeta{
+				FieldType: odatasql.PrimitiveFieldType,
+			},
+			"stateMessage": odatasql.FieldMeta{
+				FieldType: odatasql.PrimitiveFieldType,
+			},
+			"stateReason": odatasql.FieldMeta{
+				FieldType: odatasql.PrimitiveFieldType,
+			},
+			"estimation": odatasql.FieldMeta{
+				FieldType:           odatasql.ComplexFieldType,
+				ComplexFieldSchemas: []string{"Estimation"},
+			},
+			"scanTemplate": odatasql.FieldMeta{
+				FieldType:           odatasql.ComplexFieldType,
+				ComplexFieldSchemas: []string{"ScanTemplate"},
+			},
+			"summary": odatasql.FieldMeta{
+				FieldType:           odatasql.ComplexFieldType,
+				ComplexFieldSchemas: []string{"ScanEstimationSummary"},
+			},
+		},
+	},
+	"Estimation": {
+		Fields: odatasql.Schema{
+			"time": odatasql.FieldMeta{FieldType: odatasql.PrimitiveFieldType},
+			"size": odatasql.FieldMeta{FieldType: odatasql.PrimitiveFieldType},
+			"cost": odatasql.FieldMeta{FieldType: odatasql.PrimitiveFieldType}, // TODO are we handling float numbers in the db?
```

Comment:
As the data is stored in json all numbers in a json document are float64 once marshalled.